apps/right-to-rent-check/acceptance/features/person-in-property.js

```diff
@@ -21,15 +21,15 @@ Scenario('I see the correct fields on the page', (
   ]);
 });
 
-Scenario('I see errors if I submit a form without selecting anything', (
+Scenario('WHEN I submit a form without selecting anything THEN I see errors', (
   I,
   personInPropertyPage
 ) => {
   I.submitForm();
   I.seeErrors(personInPropertyPage.livingStatusGroup);
 });
 
-Scenario('I am taken to tenancy start step if I select Yes', (
+Scenario('WHEN I select Yes THEN I am taken to tenancy start page', (
```

Comment:
Also, in the previous test you make a reference to your interaction with the fields AND the form, i.e. `I submit without selecting...`, but here you only make a reference to your interaction with the field i.e `I select Yes...`